omc3/utils/iotools.py

```diff
@@ -245,7 +245,7 @@ def __new__(cls, value):
         return Path(value)
 
 
-def convert_paths_in_dict_to_strings(dict_):
+def convert_paths_in_dict_to_strings(dict_: dict) -> dict:
```

Comment:
Beware that this acts on the `dict_` inplace too.

Maybe make a copy first, or simply a temp list / dict that you return in the end.

Then the function will be pure, which we should always strive for.
Type hinting could maybe be improved?

Reply:
Changed it to not return anything and put warning in doc

Reply:
I'd rather make the functions pure honestly.

Anything goes wrong in the function and at least you've still got your initial data.
@lmalina thoughts?